crates/ruma-html/tests/it/html/matrix.rs

```diff
@@ -78,6 +79,9 @@ fn span_attributes() {
     // Uses the first spoiler attribute, the second is dropped.
     assert!(span.spoiler.unwrap().is_empty());
 
+    #[cfg(feature = "unstable-msc4286")]
+    assert_eq!(span.external_payment_details.unwrap().as_ref(), "foo");
+
     assert!(span_element.attrs.is_empty());
```

Comment:
This assertion will also fail if the cargo feature is not enabled.

Reply:
I've pushed a fix for this. (For some reason I had it in my head that unrecognised attributes were removed entirely)